src/overlays/actors/ovl_En_Gm/z_en_gm.c

```diff
@@ -20,21 +20,102 @@ void EnGm_Draw(Actor* thisx, GlobalContext* globalCtx);
+    /* 0x0115 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(13, 25, 21, 5, 0x0128 - 0x011B),
+    /* 0x011B */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(21, 55, 5, 5, 0x0122 - 0x0121),
+    /* 0x0121 */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x0122 */ SCHEDULE_CMD_RET_TIME(21, 55, 5, 5, 5),
+    /* 0x0128 */ SCHEDULE_CMD_RET_TIME(13, 25, 21, 5, 3),
+    /* 0x012E */ SCHEDULE_CMD_RET_TIME(13, 0, 13, 25, 30),
+    /* 0x0134 */ SCHEDULE_CMD_RET_TIME(12, 55, 13, 0, 20),
+    /* 0x013A */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x013B */ SCHEDULE_CMD_CHECK_DAY_L(2, 0x01EB - 0x0140),
+    /* 0x0140 */ SCHEDULE_CMD_CHECK_FLAG_L(0x32, 0x01, 0x01CD - 0x0145),
+    /* 0x0145 */ SCHEDULE_CMD_CHECK_SCENE_S(SCENE_YADOYA, 0x017A - 0x0149),
+    /* 0x0149 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(6, 0, 18, 0, 0x0174 - 0x014F),
+    /* 0x014F */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(18, 0, 21, 0, 0x016E - 0x0155),
+    /* 0x0155 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(21, 0, 21, 10, 0x0168 - 0x015B),
+    /* 0x015B */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(21, 10, 21, 15, 0x0162 - 0x0161),
+    /* 0x0161 */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x0162 */ SCHEDULE_CMD_RET_TIME(21, 10, 21, 15, 15),
+    /* 0x0168 */ SCHEDULE_CMD_RET_TIME(21, 0, 21, 10, 24),
+    /* 0x016E */ SCHEDULE_CMD_RET_TIME(18, 0, 21, 0, 6),
+    /* 0x0174 */ SCHEDULE_CMD_RET_TIME(6, 0, 18, 0, 7),
+    /* 0x017A */ SCHEDULE_CMD_CHECK_SCENE_S(SCENE_TOWN, 0x01A3 - 0x017E),
+    /* 0x017E */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(21, 10, 21, 15, 0x019D - 0x0184),
+    /* 0x0184 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(21, 15, 21, 25, 0x0197 - 0x018A),
+    /* 0x018A */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(21, 25, 21, 30, 0x0191 - 0x0190),
+    /* 0x0190 */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x0191 */ SCHEDULE_CMD_RET_TIME(21, 25, 21, 30, 14),
+    /* 0x0197 */ SCHEDULE_CMD_RET_TIME(21, 15, 21, 25, 23),
+    /* 0x019D */ SCHEDULE_CMD_RET_TIME(21, 10, 21, 15, 13),
+    /* 0x01A3 */ SCHEDULE_CMD_CHECK_SCENE_S(SCENE_MILK_BAR, 0x01CC - 0x01A7),
+    /* 0x01A7 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(21, 30, 21, 45, 0x01C6 - 0x01AD),
+    /* 0x01AD */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(21, 45, 22, 10, 0x01C0 - 0x01B3),
+    /* 0x01B3 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(22, 10, 6, 0, 0x01BA - 0x01B9),
+    /* 0x01B9 */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x01BA */ SCHEDULE_CMD_RET_TIME(22, 10, 6, 0, 5),
+    /* 0x01C0 */ SCHEDULE_CMD_RET_TIME(21, 45, 22, 10, 30),
+    /* 0x01C6 */ SCHEDULE_CMD_RET_TIME(21, 30, 21, 45, 20),
+    /* 0x01CC */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x01CD */ SCHEDULE_CMD_CHECK_SCENE_S(SCENE_YADOYA, 0x01EA - 0x01D1),
+    /* 0x01D1 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(6, 0, 18, 0, 0x01E4 - 0x01D7),
+    /* 0x01D7 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(18, 0, 6, 0, 0x01DE - 0x01DD),
+    /* 0x01DD */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x01DE */ SCHEDULE_CMD_RET_TIME(0, 0, 0, 0, 8),
+    /* 0x01E4 */ SCHEDULE_CMD_RET_TIME(6, 0, 18, 0, 7),
+    /* 0x01EA */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x01EB */ SCHEDULE_CMD_RET_NONE(),
 };
```

Comment:
```suggestion
    /* 0x1E4 */ SCHEDULE_CMD_RET_TIME(6, 0, 18, 0, 7),
    /* 0x1EA */ SCHEDULE_CMD_RET_NONE(),
    /* 0x1EB */ SCHEDULE_CMD_RET_NONE(),
};
```
There seems to be 1 too many 0's in this